libctru/source/services/apt.c

```diff
@@ -932,6 +933,7 @@ void aptLaunchSystemApplet(NS_APPID appId, void* buf, size_t bufsize, Handle han
 	aptWaitForWakeUp(TR_SYSAPPLET);
 	memcpy(buf, aptParameters, bufsize);
 	aptSetSleepAllowed(sleep);
+	aptDspWakeup();
```

Comment:
Likewise, I believe this call should be removed as in aptJumpToHomeMenu, because it is part of aptWaitForWakeUp.

Reply:
see above